src/ICSharpCode.SharpZipLib/Zip/ZipEntry.cs

```diff
@@ -1013,6 +1013,26 @@ internal int AESOverheadSize
 			}
 		}
 
+		/// <summary>
+		/// Number of extra bytes required to hold the encryption header fields.
+		/// </summary>
+		internal int EncryptionOverheadSize
+		{
+			get
+			{
+				// Entry is not encrypted - no overhead
+				if (!this.IsCrypted)
+					return 0;
+
+				// Entry is encrypted using ZipCrypto
+				if (_aesEncryptionStrength == 0)
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/piksel/SharpZipLib/pullRequest?prid=5745474)